server/plugin/command.go

```diff
@@ -190,28 +200,33 @@ func (p *Plugin) handleMuteAdd(args *model.CommandArgs, username string, userInf
 		mutedUsers = username
 	}
 
-	_, err := p.store.Set(userInfo.UserID+"-muted-users", []byte(mutedUsers))
+	_, err = p.store.Set(userInfo.UserID+"-muted-users", []byte(mutedUsers))
 	if err != nil {
 		return "Error occurred saving list of muted users"
 	}
 
 	return fmt.Sprintf("`%v`", username) + " is now muted. You'll no longer receive notifications for comments in your PRs and issues."
 }
 
-func (p *Plugin) handleUnmute(args *model.CommandArgs, username string, userInfo *GitHubUserInfo) string {
-	mutedUsernames := p.getMutedUsernames(userInfo)
+func (p *Plugin) handleUnmute(_ *model.CommandArgs, username string, userInfo *GitHubUserInfo) string {
+	mutedUsernames, err := p.getMutedUsernames(userInfo)
+	if err != nil {
+		p.client.Log.Error("error occurred getting muted user.", "Error", err)
```

Comment:
```suggestion
		p.client.Log.Error("error occurred getting muted users.", "Error", err.Error())
```
Also, add the user ID